cfme/tests/middleware/test_middleware_deployment.py

```diff
@@ -1,33 +1,48 @@
-from random import sample
-
 import pytest
+from cfme.middleware import get_random_list
 from cfme.middleware.deployment import MiddlewareDeployment
+from cfme.middleware.server import MiddlewareServer
 from utils import testgen
 from utils.version import current_version
 
 pytestmark = [
-    pytest.mark.usefixtures('single_middleware_provider'),
+    pytest.mark.usefixtures('setup_provider'),
```

Comment:
question: from where we have 'provider_name', if tests give us default provider? can we rely on UI?